lms/djangoapps/class_dashboard/tests/test_views.py

```diff
@@ -88,7 +88,8 @@ def test_section_problem_grade_distribution_no_access(self, has_access):
 
         self.assertEqual("{\"error\": \"Access Denied: User does not have access to this course\'s data\"}", response.content)
 
-    def test_sending_deprecated_id(self):
+    @patch('class_dashboard.dashboard_data.settings')
+    def test_sending_deprecated_id(self, mock_settings):
```

Comment:
I guess my questions was why the settings had to be _mocked_.  Tests also run with a set of django settings (lms/envs/test.py), but I guess those weren't sufficient?